client/src/cli/apiToken.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  AccessTokenError,
+  getRmmbrAccessTokenPath,
+} from "./rmmbrAccessToken.ts";
+import { err, ok, Result } from "./deps.ts";
+
+const rmmbrServer = Deno.env.get("RMMBR_SERVER");
+
+export const apiToken = async () => {
+  const result: Result<string, AccessTokenError> = await (
+    await getAccessToken()
+  ).match({
+    Err: async (e) => await err(e),
+    Ok: async (accessToken) => await getOrCreateApiToken(accessToken),
```

Comment:
`Ok: getOrCreateApiToken`

Reply:
This doesn't work for some reason... try it for yourself, not sure why. I can however remove `async` and `await`.